<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册账号</string>
  </property>
  <widget class="QGroupBox" name="gbRegister">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>350</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>由数字和字母组成</string>
   </property>
   <property name="title">
    <string>注册账号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="lblPhone">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>请输入您的电话号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPW">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请设置您的密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblState1">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblState2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>100</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePhone">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="placeholderText">
     <string>11位手机号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePW">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>由数字和字母组成</string>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>8位密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>290</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblErrMsg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPWDeux">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请确认您的密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblState3">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>150</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePWDeux">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>由数字和字母组成</string>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>8位密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择您的身份：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbChar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lePhone</tabstop>
  <tabstop>lePW</tabstop>
  <tabstop>lePWDeux</tabstop>
  <tabstop>cbChar</tabstop>
  <tabstop>btnSubmit</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>RegisterForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSubmit</sender>
   <signal>clicked()</signal>
   <receiver>RegisterForm</receiver>
   <slot>SubmitReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbChar</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RegisterForm</receiver>
   <slot>BindSelect(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterForm</receiver>
   <slot>CheckPhone(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePW</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterForm</receiver>
   <slot>CheckPW(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePWDeux</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisterForm</receiver>
   <slot>CheckDeux(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CheckPhone(QString)</slot>
  <slot>CheckPW(QString)</slot>
  <slot>SubmitReg()</slot>
  <slot>CheckDeux(QString)</slot>
  <slot>BindSelect(int)</slot>
 </slots>
</ui>
